pybaseball/statcast_fielding.py

```diff
@@ -36,6 +36,30 @@ def statcast_outs_above_average(year: int, pos: Union[int, str], min_att: Union[
 	data = sanitize_statcast_columns(data)
 	return data
 
+@cache.df_cache()
+def statcast_fielding_run_value(year: int, pos: Union[int, str], min_inn: int = 100) -> pd.DataFrame:
+	"""Scrapes fielding run value from baseball savant for a given year and position
+
+	Args:
+		year (int): Season to pull
```

Comment:
If `year=0` we could get player's merged data. Maybe we could also mention this option
ref: https://baseballsavant.mlb.com/leaderboard/fielding-run-value?year=0&team=&min=0.1&pos=

FYI: We could get year split data with `year=split` but maybe we won't need it for now